src/components/TokenDisplay.tsx

```diff
@@ -25,29 +20,9 @@ interface Props {
   testID?: string
 }
 
-function calculateDecimalsToShow(value: BigNumber) {
-  const exponent = value?.e ?? 0
-  if (exponent >= 0) {
-    return DEFAULT_DISPLAY_DECIMALS
-  }
-
-  return Math.abs(exponent) + 1
-}
-
-// Formats |value| so that it shows at least 2 significant figures and at least 2 decimal places without trailing zeros.
-export function formatValueToDisplay(value: BigNumber) {
```

Comment:
this should eventually exist in this file right? When LegacyTokenDisplay usages are removed and cleaned up